ctapipe/reco/tests/test_event_classifier.py

```diff
@@ -1,6 +1,81 @@
 from tempfile import TemporaryDirectory
 from ctapipe.reco.event_classifier import *
+from sklearn.neural_network import MLPClassifier
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import StandardScaler
+import pytest
 
+@pytest.mark.xfail(run=False, reason="Can't pass Pipeline as a classifier "
+                                     "to actual implementation of RegressorClassifierBase")
+def test_pipeline_classifier():
+    cam_id_list = ["FlashCam", "ASTRICam"]
+    feature_list = {"FlashCam": [[1, 10], [2, 20], [3, 30], [0.9, 9],
+                                 [10, 1], [20, 2], [30, 3], [9, 0.9]],
+                    "ASTRICam": [[10, 1], [20, 2], [30, 3], [9, 0.9],
+                                 [1, 10], [2, 20], [3, 30], [0.9, 9]]}
+    target_list = {"FlashCam": ["a", "a", "a", "a", "b", "b", "b", "b"],
+                   "ASTRICam": ["a", "a", "a", "a", "b", "b", "b", "b"]}
+
+    estimators = [('scaler', StandardScaler()), ('clf', MLPClassifier())]
+
+    clf = EventClassifier(classifier=Pipeline(estimators), cam_id_list=cam_id_list,
```

Comment:
`classifier` expects a class-name not an already initialised instance. You need to use
`EventClassifier(classifier=Pipeline, steps=estimators …`
and then you need to modify the line that threw the previous error in `regressor_classifier_base.py`
to `self.model_dict[cam_id] = model(**deepcopy(kwargs))` (of course import deepcopy first). Otherwise, every camera_id uses the same model and we wouldn't want that ;)